### PR TITLE
create pdf out of the documentation

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by whummer — APPROVED** [review]
Looks great, kudos for putting this together so quickly @HarshCasper ! 🚀 

Added a few tiny comments, nothing that should block the merge..

Small note: seems like the border on the PDF pages is pretty small, hence the content sits very much at the top of the page (at least in my PDF reader). No need to change it for now though, could be done in a future iteration. 👍 

Example 1:
image

Example 2:
image

File: scripts/generate_pdf_output.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        "tags",
+        "categories",
+        "applications",
+        "references/coverage",
+    ]
+    delete_folders(folders_to_delete)
+    source_directory = "w2pdf_output"
+    target_directory = "final"
+    find_and_copy_pdfs(source_directory, target_directory)
+    merge_pdfs(doc_list, "localstack_docs.pdf")
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply scripts/generate_pdf_output.py:136]: Yeah, will follow up on the other PR 👍

File: scripts/pdf_list.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    'gitlab-ci',
+    'harness-ci',
+    'ci-keys',
+    'localstack-extensions',
+    'getting-started',
+    'managing-extensions',
+    'developing-extensions',
+    'official-extensions',
+    'cloud-pods',
+    'getting-started',
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply scripts/pdf_list.py:130]: Seems like a bug. Will fix later in follow-up PR

File: scripts/generate_pdf_output.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                shutil.rmtree(folder_path)
+            else:
+                print(f"Folder not found: {folder_path}")
+        except OSError as e:
+            print(f"Error deleting folder {folder_path}: {e}")
+
+
+if __name__ == "__main__":
+    run_website2pdf()
+    time.sleep(60)
```
[the remark this answers is not in the document]
Comment (HarshCasper) [reply scripts/generate_pdf_output.py:122]: Yeah, this is required to setup all the PDF files before processing and merged into a final PDF

**Pull request merged**